### PR TITLE
add missing dependency for schema generation view

### PR DESCRIPTION
**Pull request by felixrindt**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,6 +43,7 @@ redis = {version = "^4.3.4", optional = true}
 django-oauth-toolkit = "^2.3.0"
 mozilla-django-oidc = "^3.0.0"
 urllib3 = "^1.26.0,<2.0.0"  # pinned because of poetry issues with urllib3 2.0.0
+pyyaml = "^6.0.1"
```
Comment (jeriox) [inline pyproject.toml:46]: why is this required? is this just a missing dependency in another package?

File: ephios/settings.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 DEBUG = env.bool("DEBUG")
 ALLOWED_HOSTS = env.list("ALLOWED_HOSTS")
 
-EPHIOS_VERSION = "v" + importlib_metadata.version("ephios")
+EPHIOS_VERSION = importlib_metadata.version("ephios")
```
[the remark this answers is not in the document]
Comment (felixrindt) [reply ephios/settings.py:35]: Yes. I thought about adding it in the footer template but I think it's also fine without isn't it?

File: pyproject.toml (hunk 1 above)
Comment (felixrindt) [reply pyproject.toml:46]: It's required for the SchemaView of drf. See the documentation there...

Comment (felixrindt) [reply pyproject.toml:46]: image
https://www.django-rest-framework.org/api-guide/schemas/

File: ephios/settings.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeriox) [reply ephios/settings.py]: i think i like it shlighty more with the v, but no hard feelings

File: pyproject.toml (hunk 1 above)
Comment (jeriox) [reply pyproject.toml:46]: wow. still think that it _should_ be their job to specify that ^^

**Review by jeriox — APPROVED** [review]

File: ephios/settings.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (felixrindt) [reply ephios/settings.py]: You know I was thinking reading the footer flows better with the v there. Otherwise your mind has to add the "version" when reading. I'll add it back in...

File: pyproject.toml (hunk 1 above)
Comment (felixrindt) [reply pyproject.toml:46]: well its really an optional feature but yeah
